unstructured/unstructured.go

```diff
@@ -0,0 +1,103 @@
+package unstructured
+
+import (
+	"errors"
+)
+
+// MissingField is the error returned when the requested field could not
+// be found.
+var MissingField = errors.New("field not found")
```

Comment:
https://dave.cheney.net/2016/04/27/dont-just-check-errors-handle-them-gracefully

https://dave.cheney.net/2014/12/24/inspecting-errors

Reply:
Great reading, thanks for the pointer. I'll go with the "opaque" error, and just not test the specific type. I think that's fine. Exporting less sounds good.